srcpkgs/ovmf-ia32/template

```diff
@@ -0,0 +1,118 @@
+# Template file for 'ovmf-ia32'
+pkgname=ovmf-ia32
```

Comment:
I don't really like the invented ppackage name at the toplevel, can we just do it like alpine and package edk2 and make the ovmf stuff subpackages?